pkg/commands/patch/patch_manager.go

```diff
@@ -192,21 +195,15 @@ func (p *PatchManager) RenderPatchForFile(filename string, plain bool, reverse b
 
 func (p *PatchManager) renderEachFilePatch(plain bool) []string {
 	// sort files by name then iterate through and render each patch
-	filenames := make([]string, len(p.fileInfoMap))
-	index := 0
-	for filename := range p.fileInfoMap {
-		filenames[index] = filename
-		index++
-	}
+	filenames := maps.Keys(p.fileInfoMap)
 
 	sort.Strings(filenames)
-	output := []string{}
-	for _, filename := range filenames {
-		patch := p.RenderPatchForFile(filename, plain, false, true)
-		if patch != "" {
-			output = append(output, patch)
-		}
-	}
+	patches := slices.Map(filenames, func(filename string) string {
+		return p.RenderPatchForFile(filename, plain, false, true)
+	})
+	output := slices.Filter(patches, func(patch string) bool {
+		return patch != ""
+	})
```

Comment:
interesting case where you could use a filter-map but I find this reads a little better